tests/integration/cli/test_compare.py

```diff
@@ -179,8 +181,9 @@ def test_can_compare_projects(self):  # just a smoke test
 
             self.assertNotEqual(0, os.listdir(osp.join(test_dir)))
 
+    @patch("sys.stdout", new_callable=io.StringIO)
     @mark_requirement(Requirements.DATUM_GENERAL_REQ)
-    def test_can_run_distance_diff(self):
+    def test_can_run_distance_diff(self, mock_stdout):
```

Comment:
Best practice for capturing `stdout` or `stderr` is using a special fixture provided by `pytest`. Please see https://pytest.org/en/7.4.x/how-to/capture-stdout-stderr.html#accessing-captured-output-from-a-test-function